src/_P039_Thermosensors.ino

```diff
@@ -760,8 +790,8 @@ float readMax6675(struct EventStruct *event)
     // Shift RAW value 3 Bits to the right to get the data
     rawvalue >>= 3;
 
-    // Calculate Celsius
-    return rawvalue * 0.25f;
+    // Calculate Celsius with device resolution 0.25 °K/bit
```

Comment:
OK, now we enter the area of nitpicking...
In Kelvin you don't use degree as you would on Celcius and Fahrenheit scale.

Reply:
I can follow the remark - even more disturbing to me would be the missing brackets around the return value - already fixed in one of my next pushes.

Reply:
You don't need the () around the value for the return.
This isn't C you know :)

Reply:
Some C habits will remain with my programming...

Reply:
Solved in upcoming push !